src/ISA/ISA/ArcTypes/ArcInvestigation.fs

```diff
@@ -266,4 +266,37 @@ type ArcInvestigation(identifier : string, ?title : string, ?description : strin
         )
 
 
-    
\ No newline at end of file
+    /// Transform an ArcInvestigation to an ISA Json Investigation.
+    member this.ToStudy() : Investigation = 
```

Comment:
This is wrong?

Reply:
Should these be type extensions in a different name space ?

Reply:
I thought about it. But both the `ArcAssay` and the `Assay` are part of the same namespace, so I guess the transformation between them should also be there.

Reply:
Bu what do we except the developer user to see? The only reason why we keep the json types in the main namespace are the smaller subtypes such as OntologyAnnotation. For now keep it there.